src/store/reducers/storage/selectors.ts

```diff
@@ -72,6 +77,38 @@ export const selectNodesUptimeFilter = (state: StorageStateSlice) =>
     state.storage.nodesUptimeFilter;
 export const selectStorageType = (state: StorageStateSlice) => state.storage.type;
 
+// ==== Sort params selectors ====
+export const selectNodesSortParams = (state: StorageStateSlice) => {
+    const defaultSortValue: NodesSortValue = NODES_SORT_VALUES.NodeId;
+    const defaultSortOrder: OrderType = ASCENDING;
+
+    return {
+        sortValue: state.storage.nodesSortValue || defaultSortValue,
+        sortOrder: state.storage.nodesSortOrder || defaultSortOrder,
+    };
```

Comment:
These default values previously was set in `StorageNodes`.

Removed sorting by `Degraded` disks since this values cannot be sorted via request params